ui/raidboss/data/07-dt/raid/r7n.txt

```diff
@@ -1,9 +1,274 @@
+80.7 "Quarry Swamp" Ability { id: "A52D", source: "Brute Abombinator" }
+92.7 "Explosion" Ability { id: "A52E", source: "Brute Abombinator" }
+95.3 "Explosion" #Ability { id: "A52E", source: "Brute Abombinator" }
+97.7 "Explosion" #Ability { id: "A52E", source: "Brute Abombinator" }
+97.7 "Stoneringer: The Stabbing" Ability { id: "A51B", source: "Brute Abombinator" }
+107.6 "Brutish Swing" Ability { id: "A51D", source: "Brute Abombinator" }
+108.6 "Brutish Swing" Ability { id: "A51F", source: "Brute Abombinator" }
+109.6 "Brutal Smash" Ability { id: "A521", source: "Brute Abombinator" }
+114.7 "--sync--" Ability { id: "A516", source: "Brute Abombinator" }
+124.8 "Neo Bombarian Special" Ability { id: "A52F", source: "Brute Abombinator" }
+
+# Second arena
+135.2 "Grappling Ivy" Ability { id: "A530", source: "Brute Abombinator" }
+140.2 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+143.2 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+147.2 "Stoneringer: The Bludgeoning" Ability { id: "A532", source: "Brute Abombinator" }
+156.1 "Brutish Swing" Ability { id: "A534", source: "Brute Abombinator" }
+157.1 "Brutish Swing" Ability { id: "A535", source: "Brute Abombinator" }
+158.1 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+163.1 "Stoneringer: The Stabbing" Ability { id: "A533", source: "Brute Abombinator" }
+173.0 "Brutish Swing" Ability { id: "A539", source: "Brute Abombinator" }
+175.9 "Brutish Swing" Ability { id: "A541", source: "Brute Abombinator" }
+177.1 "Brutish Swing" Ability { id: "A53F", source: "Brute Abombinator" }
+181.7 "Glower Power" Ability { id: "A546", source: "Brute Abombinator" }
+182.9 "Glower Power" Ability { id: "A94B", source: "Brute Abombinator" }
+183.0 "Electrogenetic Force" Ability { id: "A547", source: "Brute Abombinator" }
+190.0 "Abominable Blink" Ability { id: "A542", source: "Brute Abombinator" }
+191.0 "Abominable Blink" Ability { id: "A892", source: "Brute Abombinator" }
+192.0 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+199.0 "Revenge of the Vines" Ability { id: "A543", source: "Brute Abombinator" }
+201.0 "--sync--" Ability { id: "A893", source: "Brute Abombinator" }
+206.0 "Stoneringer: The Stabbing" Ability { id: "A533", source: "Brute Abombinator" }
+216.9 "Sporesplosion" Ability { id: "A544", source: "Brute Abombinator" }
+223.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+225.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+225.4 "Brutish Swing" Ability { id: "A53D", source: "Brute Abombinator" }
+227.0 "Sporesplosion" #Ability { id: "A545", source: "Brute Abombinator" }
+228.3 "Brutish Swing" Ability { id: "A541", source: "Brute Abombinator" }
+229.5 "Brutish Swing" Ability { id: "A53F", source: "Brute Abombinator" }
+234.1 "Glower Power" Ability { id: "A546", source: "Brute Abombinator" }
+235.3 "Glower Power" Ability { id: "A94B", source: "Brute Abombinator" }
```

Comment:
It's probably fine for this PR, but we might want to add a TODO for "clean up the castbars vs in/out vs stack/tank stack" here and elsewhere